### PR TITLE
{bio}[system/system] CNS v1.31

### PR DESCRIPTION
**Pull request by jordiblasco**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@jordiblasco Can you provide a test report for this?

File: easybuild/easyconfigs/c/CNS/CNS-1.31-linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Binary'
+
+name = 'CNS'
+version = '1.31'
```
Comment (smoors) [inline easybuild/easyconfigs/c/CNS/CNS-1.31-linux-x86_64.eb:11]: @jordiblasco I was only able to see version 1.3 after filling the request for, are you sure this version is correct?

Comment (jordiblasco) [reply easybuild/easyconfigs/c/CNS/CNS-1.31-linux-x86_64.eb:11]: Yes, that's the version I downloaded after filling up the request form: http://cns-online.org/cns_request/

Comment (jordiblasco) [reply easybuild/easyconfigs/c/CNS/CNS-1.31-linux-x86_64.eb:11]: Yes, that was the version I got after filling up the request form: http://cns-online.org/cns_request/

**Comment by jordiblasco** [comment]
HADDOCK depends on this software: https://github.com/easybuilders/easybuild-easyconfigs/pull/11689

**Comment by boegel** [comment]
closing this: no test report, was only needed for #11689 which was closed a while ago already

**Pull request closed without merging**